pymongo/command_cursor.py

```diff
@@ -51,7 +71,7 @@ def __init__(
         self.__collection: Collection[_DocumentType] = collection
         self.__id = cursor_info["id"]
         self.__data = deque(cursor_info["firstBatch"])
-        self.__postbatchresumetoken = cursor_info.get("postBatchResumeToken")
+        self.__postbatchresumetoken: int = cast(int, cursor_info.get("postBatchResumeToken"))
```

Comment:
postBatchResumeToken should be an optional mapping, not an int.

Reply:
Changed!